### PR TITLE
Rk/add local warmup

### PR DESCRIPTION
**Pull request by przemekak**

### CODE REVIEW

File: benchto-driver/src/main/java/io/trino/benchto/driver/execution/BenchmarkExecutionDriver.java [hunk 1]
```diff
@@ -380,6 +392,7 @@ private QueryExecutionResult executeSingleQuery(
             Optional<Path> outputFile)
             throws TimeLimitException
     {
+        LOG.info("Execute query, query=%s, skipReport=%s".formatted(benchmark.getQueries().get(0).getName(), Boolean.toString(skipReport)));
```
Comment (radek-starburst) [inline benchto-driver/src/main/java/io/trino/benchto/driver/execution/BenchmarkExecutionDriver.java:395]: Do we actually need that?

**Review by radek-starburst — COMMENTED** [review]
Maybe add a test that directly tests `local warmup`?

**Review by radek-starburst — APPROVED** [review]

File: benchto-driver/src/main/java/io/trino/benchto/driver/execution/BenchmarkExecutionDriver.java (hunk 1 above)
Comment (przemekak) [reply benchto-driver/src/main/java/io/trino/benchto/driver/execution/BenchmarkExecutionDriver.java:395]: > Do we actually need that?

It actually is nice when you can see in logs what query is executed in warmup phase and in normal measurement phase (skipReport=false)

**Comment by przemekak** [comment]
> Maybe add a test that directly tests `local warmup`?

Default is set to 2, so all tests e.g.  in `DriverAppIntegrationTest` are running 2 local local warmups and we are checking number of executions. Should be enough?

File: benchto-driver/src/main/java/io/trino/benchto/driver/execution/BenchmarkExecutionDriver.java (hunk 1 above)
Comment (sopel39) [reply benchto-driver/src/main/java/io/trino/benchto/driver/execution/BenchmarkExecutionDriver.java:395]: I don't think you need to call `Boolean.toString(`

**Review by sopel39 — APPROVED** [review]

**Comment by sopel39** [comment]
small comments, please also make sure it works as expected in real setup.

**Pull request merged**